src/shell/commands.cpp

```diff
@@ -4,7 +4,7 @@
 
 static void moveCallback() {
 	delay(200);
-	motion->enable(false);
+	//motion->enable(false);
```

Comment:
I think it would be better to create a dedicated command to trigger a recalibration.